docs/source/conf.py

```diff
@@ -27,11 +27,28 @@
 from sphinx_gallery.sorting import ExplicitOrder
 
 TMP_FOLDER = Path(__file__).parents[2] / "tmp"
+TMP_FOLDER.mkdir(exist_ok=True)
+
+# Generate a JSON compatibility table
+
+html_extra_path = []
+
+with open("additional/releases.yaml") as f:
+    table = yaml.safe_load(f)
```

Comment:
Since it's not just used in this context anymore, I realize `table` isn't a very good name, maybe `releases` or something like that would be better.

Reply:
Ok good idea :)